packages/image_picker/image_picker_android/example/android/app/src/main/java/io/flutter/plugins/imagepickerexample/DummyContentProvider.java

```diff
@@ -0,0 +1,68 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.imagepickerexample;
+
+import android.content.ContentProvider;
+import android.content.ContentValues;
+import android.content.res.AssetFileDescriptor;
+import android.database.Cursor;
+import android.database.MatrixCursor;
+import android.net.Uri;
+import android.provider.MediaStore;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+public class DummyContentProvider extends ContentProvider {
+  @Override
+  public boolean onCreate() {
+    return true;
+  }
+
+  @Nullable
+  @Override
+  public AssetFileDescriptor openAssetFile(@NonNull Uri uri, @NonNull String mode) {
+    return getContext().getResources().openRawResourceFd(R.raw.flutter_logo);
+  }
+
+  @Nullable
+  @Override
+  public Cursor query(
+      @NonNull Uri uri,
+      @Nullable String[] projection,
+      @Nullable String selection,
+      @Nullable String[] selectionArgs,
+      @Nullable String sortOrder) {
+    MatrixCursor cursor = new MatrixCursor(new String[] {MediaStore.MediaColumns.DISPLAY_NAME});
+    cursor.addRow(new Object[] {"dummy.png"});
+    return cursor;
+  }
+
+  @Nullable
+  @Override
+  public String getType(@NonNull Uri uri) {
+    return "image/png";
+  }
+
+  @Nullable
+  @Override
+  public Uri insert(@NonNull Uri uri, @Nullable ContentValues values) {
+    return null;
+  }
+
+  @Override
+  public int delete(
+      @NonNull Uri uri, @Nullable String selection, @Nullable String[] selectionArgs) {
+    return 0;
+  }
+
+  @Override
+  public int update(
+      @NonNull Uri uri,
+      @Nullable ContentValues values,
+      @Nullable String selection,
+      @Nullable String[] selectionArgs) {
+    return 0;
+  }
+}
```

Comment:
This comment refers to the flutter_logo.png file

We generally don't include image files that are non-trivially large in the repo if possible. We prefer to either get them in another repo and import them or if the goal is to have an image to test with, generate a trivial image. @stuartmorgan Do we have this rule in the plugins repo? What do you think about this case?

Reply:
We don't have a specific rule/enforcement around this for flutter/plugins, since checking out the entire repo isn't part of the normal usage of plugins (unlike for flutter/flutter). But using a copy of one of the images that's already in the repo, such as an app icon image from one of the example templates, would be preferable to a new image.

Reply:
We cannot easily use an existing file, because it needs to be placed in the 'res/raw' folder. But I replaced this logo file with a copy of the smallest icon, just ~500 bytes so it shouldn't be a problem.